packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -506,6 +541,12 @@ class OffsetLayer extends ContainerLayer {
   /// pipeline.
   Offset offset;
 
+  @override
+  S find<S>(Offset regionOffset) {
+    final Offset transformed = regionOffset - offset;
+    return super.find<S>(transformed);
```

Comment:
i don't mind which we do, but we should be consistent in the style between this one and LeaderLayer's.

Reply:
Fixed